### DISCUSSION TITLE
Not Working on Windows 11 VMWare, SEB v3.4.1.505

### DISCUSSION THREAD

**Discussion by bryanc12** [nxvvvv/safe-exam-browser-bypass#1] [General]
![image](https://user-images.githubusercontent.com/34401335/228022899-44dcd0aa-bff8-4242-b0dc-475d2fd6a3bf.png)

**Comment by nxvvvv** [comment]
Haven't updated it in a while. That's why

**Reply by Zwilgmeyer** [reply]
Any plans to make a new version? You would be doing the lords work

**Comment by Ta-LT** [comment]
Please update, My Lord!

**Comment by marcstae** [comment]
It would be really nice if you could update this repo or give a hint on where I should make changes to get this working again. :)
Thank you

**Comment by Hunais-Abdla** [comment]
Anyway for SEB lite to work ok windows 11 Via VMware on apple silicon

**Comment by Nevydaylo** [comment]
How i do the install on MAC ?

**Comment by Hunais-Abdla** [comment]
Anyway for SEB lite to work ok windows 11 Via VMware on apple silicon

**Reply by nxvvvv** [reply]
Try the new updated SEB, it might work. I'm not sure about SEB Lite tho

**Comment by nxvvvv** [comment]
@bryanc12 @Ta-LT @marcstae @HunAbdla @Nevydaylo @dritogdraUIS Updated the Client. Try it out
